code/src/components/MovieDetails/MovieDetails.css

```diff
@@ -0,0 +1,112 @@
+.background-image {
```

Comment:
You have different approaches on the styling, in MovieDetails you have styled in the returs statement, only the linar-gradient shadow, but not something else. Were there a specific reason for that? Also the size for the circle.

Reply:
We wanted to have the gradient in the picture. @brunsant tried doing the overlay in the CSS file, but the only way it worked was as inline styling. 
When it comes to the circle, it was an icon from the `react-icon` library. These icons only allow for inline styling.